### ISSUE TITLE
Suggestion: #button:delete

### ISSUE THREAD

**Issue by khoogheem** [nodes-vapor/admin-panel-provider#31]
Currently the LeafTag #button:delete forces a sr-only of Delete User.

Suggestion to change it to:
```
Delete \(title)
```
Or add an option third option for the sr-only

**Comment by steffendsommer** [comment]
I agree. Also we should consider making the class of the button a bit more flexible. Thanks for reporting this.

**Comment by khoogheem** [comment]
If you want.. I had started up a Bootstrap Leaf Provider.  Might make sense to look at potentially using that. 
Gets a little hairy with all of the options that can go into the html.. but tried to make at least the most common things easy.

https://bootstrap-provider.vapor.cloud
